source/ui/IDEAS/IDEASPersistentView.cpp

```diff
@@ -323,13 +340,19 @@ void IDEASPersistentView::onCalibrateClicked()
 void IDEASPersistentView::onScalerContinuousButtonClicked()
 {
 	AMListAction3 *scalerContinuousEnableActions = new AMListAction3(new AMListActionInfo3("Enable Scaler Continuous Mode", "Enable Scaler Continuous Mode"));
-	scalerContinuousEnableActions->addSubAction(new AMWaitAction(new AMWaitActionInfo(0.25)));
+	scalerContinuousEnableActions->addSubAction(new AMWaitAction(new AMWaitActionInfo(0.5)));
```

Comment:
Not necessary for this pull request, but maybe we should try looking into that again?